book/src/from-vim.md

```diff
@@ -0,0 +1,5 @@
+# Migrating from Vim
+
+Helix's editing model is strongly inspired from vim and kakoune, and a notable difference from vim (and the most striking similarity to kakoune) is that Helix follows the `object → verb` model. This means that the whatever you are going to act on (a word, a paragraph, a line, etc) comes first and the action itself (delete, change, yank, etc) comes second.
```

Comment:
A minor quibble, and something reasonable people can disagree about:

I think the `object -> verb` terminology is a bit weird, because it frames the interaction model as if it's related to linguistics.  `selection -> action` is much clearer, IMO, and keeps it squarely in the realm of interaction models.

(It also has the side-benefit of not inadvertently triggering people's irrelevant "this the opposite of English/French/Spanish/whatever-my-native-language, and therefore weird" reactions.  Most common text editors like notepad, gedit, Word, etc. actually follow the `selection -> action` model as well--albeit with a much reduced set of commands--so it's actually the norm.  But the linguistic framing obscures that.)

Reply:
That's a good point, I'll change it 👍🏾